src/main/java/core/basesyntax/DefaultOperationStrategy.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DefaultOperationStrategy implements OperationStrategy {
+    private Map<FruitTransaction.Operation, OperationHandler> handlers = new HashMap<>();
+
+    public DefaultOperationStrategy() {
+        handlers.put(FruitTransaction.Operation.BALANCE, new BalanceHandler());
+        handlers.put(FruitTransaction.Operation.SUPPLY, new SupplyHandler());
+        handlers.put(FruitTransaction.Operation.PURCHASE, new PurchaseHandler());
+        handlers.put(FruitTransaction.Operation.RETURN, new ReturnHandler());
+    }
```

Comment:
Please initialize map in Main method and pass through constructor. This will make code more maintainable and testable

```suggestion
    private Map<FruitTransaction.Operation, OperationHandler> handlers;

    public DefaultOperationStrategy(Map<FruitTransaction.Operation, OperationHandler> handlers) {
        this.handlers = handlers;
    }
```